compressor/conf.py

```diff
@@ -45,8 +45,8 @@ class CompressorConf(AppConf):
     YUGLIFY_BINARY = 'yuglify'
     YUGLIFY_CSS_ARGUMENTS = '--terminal'
     YUGLIFY_JS_ARGUMENTS = '--terminal'
-    CLEAN_CSS_BINARY = 'cleancss'
-    CLEAN_CSS_ARGUMENTS = ''
+    COMPRESS_CLEAN_CSS_BINARY = 'cleancss'
+    COMPRESS_CLEAN_CSS_ARGUMENTS = ''
```

Comment:
appconf transforms all properties set in that class, adding the `COMPRESS_` prefix before turning them in settings, so this is wrong.